<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Your message:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="messageLineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendMessageButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: purple;
    color: white;
	border: 2px solid black;
	border-radius: 10px;
	 padding: 10px; 
}
 QPushButton:hover {
	 background-color: green; 
	 color: white;
   }</string>
        </property>
        <property name="text">
         <string>Sent to all</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="privateMessageButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: purple;
    color: white;
	border: 2px solid black;
	border-radius: 10px;
	 padding: 10px; 
}
 QPushButton:hover {
	 background-color: green; 
	 color: white;
   }</string>
        </property>
        <property name="text">
         <string>Send private</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextBrowser" name="commonChatTextBrowser">
        <property name="styleSheet">
         <string notr="true">QTextBrowser {
	background-color: rgb(173, 216, 230);
    color: black;
	border-radius: 10px;
	 padding: 10px; 
}
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="privateChatTextBrowser">
        <property name="styleSheet">
         <string notr="true">QTextBrowser {
	background-color: rgb(173, 216, 230);
    color: black;
	border-radius: 10px;
	 padding: 10px; 
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="messageForMeButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: blue;
    color: white;
	border: 2px solid black;
	border-radius: 10px;
	 padding: 10px; 
}
 QPushButton:hover {
	 background-color: green; 
	 color: yellow;
   }</string>
      </property>
      <property name="text">
       <string> Message for me</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnecting_a_new_client"/>
    <addaction name="actionClose_client"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnecting_a_new_client">
   <property name="text">
    <string>Connecting a new client</string>
   </property>
  </action>
  <action name="actionClose_client">
   <property name="text">
    <string>Close client</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
